components/brave_new_tab_ui/apiEventsToStore.ts

```diff
@@ -9,8 +9,9 @@ import * as statsAPI from './api/stats'
 import * as topSitesAPI from './api/topSites'
 import * as privateTabDataAPI from './api/privateTabData'
 import * as newTabAdsDataAPI from './api/newTabAdsData'
-import getNTPBrowserAPI, { CustomBackground } from './api/background'
+import getNTPBrowserAPI, { Background } from './api/background'
 import { getInitialData, getRewardsInitialData, getRewardsPreInitialData } from './api/initialData'
+import * as BackgroundData from './data/backgrounds'
```

Comment:
Nit: Personally I prefer the upper case imports, but it looks like the other `* as from` imports are `lowerCamelCase`